src/zenml/log_stores/datadog/datadog_log_store.py

```diff
@@ -125,7 +136,7 @@ def fetch(
                     "page": {
                         "limit": page_limit,
                     },
-                    "sort": "@otel.timestamp",
+                    "sort": "timestamp",
```

Comment:
We have changed this on purpose because of the sorting. I know the exporter has changed. Is this handled properly now?

Reply:
yes.  the exported logs have the Datadog timestamp correctly set now, so we can use it when we fetch the entries